Python/_warnings.c

```diff
@@ -1399,12 +1399,12 @@ PyErr_WarnExplicitFormat(PyObject *category,
 }
 
 void
-_PyErr_WarnUnawaitedAgenMethod(PyAsyncGenObject *agen, PyObject *method)
+_PyErr_WarnUnawaitedAgenMethod(PyGenObject *agen, PyObject *method)
```

Comment:
This seems a less correct type. `agen` should be an async generator, so `PyAsyncGenObject *` seems the correct type.

Reply:
Are we going to keep PyAsyncGenObject and PyCoroObject, now that they are just aliases?
It's not like using them in a signature will give compile-time type checks.

I was assuming we will want to stop using them altogether and deprecate them from the API.